src/PageContent/AssignVolunteers/Edit.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import moment from 'moment';
+import { StringFormat } from '../../Enums';
+
+class Edit extends Component {
+
+    constructor(props) {
+        super(props);
+        let date = new Date(this.props.delivery.date);
+        this.state = {
+            day: moment(date).format('dddd'),
+            date: moment(date).format('l'),
+            from: this.props.delivery.startTime,
+            to: this.props.delivery.endTime,
+            donatingAgency: this.props.delivery.donatingAgency.agency,
+            receivingAgency: this.props.delivery.receivingAgency.agency,
+            deliverer1: {
+                name: '',
+                phone: '',
+                email: ''
+            },
+            deliverer2: {
+                name: '',
+                phone: '',
+                email: ''
+            }
```

Comment:
for the way you're doing now, you don't even need these `deliverer1` `deliverer2` in the `state` anymore

Reply:
Fixed.